_content/overview.md

```diff
@@ -1,100 +1,103 @@
 
 ### General principles
 
 #### Style and tone
 
-Best practices to adopt:
-
-- Plan what to write before you write.
-- Always keep your audience in mind when you write.
-- Use active voice when possible.
-- Write for scannability; use bullet points, short paragraphs, and sections/headers to break up your content.
-- Be brief.
-- Oxford comma: In a list of three or more items, add a comma before the conjunction (for example: "Android, iOS, and Windows").
-- Spell check your content.
-- Be friendly by using "you". And remember that Agora can "recommend" courses of action.
-- Review your content. Edit out any information your reader does not need to know.
-- Use parallel structures to tell the reader how related concepts are different and how they are similar.
-- Place essential information in the main clause, as you cannot assume the reader will read the whole sentence.
-- Remove ambiguity by choosing words whose meaning is clear.
-- Use the relative pronoun "that" where you might otherwise omit it, and prefer using it to a noun-...ing combination, for example say "the method that uses this parameter" instead of "the method using this parameter".
+Best practices to adopt: //TODO William Are there too many? Should we simplify this or break it up? If you wanted something someone could maybe print (on one page) and put up by their workstation, this might be too much.
+
+* Create an outline to plan what to write before you start.
+* Always keep your audience's capabilites and needs in mind when you write.
+* Write for scannability by using bullet points, short paragraphs, and informative section headings to break up your content.
+* Keep paragraphs short. Dense pages full of text can be intimidating for readers (or hide important information from skimmers).
+* Remember to be clear and concise. Replace phrases with single words where possible, and remove ambiguities.
+* Favor the active voice.
+* Maintain a formal tone (except for FAQs). //TODO WIlliam I think?
+* Write in the second person ("you"), because it is more friendly and engaging than a third-person form (such as "developers"). Do not use "we," but remember that "Agora" can recommend courses of action. 
+* Always be consistent in your word usage. Do not use the same word to mean different things (or refer to one thing using different words).
+* Place essential information in the main clause, as you cannot assume the reader will read the whole sentence.
+* Put conditional clauses first. Start by mentioning the objective, followed the instruction (for example, "To run the program, click Run.").
+* Use the Oxford comma. That is, in a list of three or more items, add a comma before the conjunction (for example. "Android, iOS, and Windows").
+* Use parallel structures to tell the reader how related concepts are different and how they are similar.
+* Vary the length of your sentences.
```

Comment:
Maybe put this together with the "Keep paragraphs short" one? Or just delete it?

Reply:
Yes, as part of the general consolidation this should get cleaned up.